jocd/src/br/org/certi/jocd/flash/FlashAlgo.java

```diff
@@ -25,8 +25,8 @@
   public int pcEraseAll;
   public int pcEraseSector;
   public int pcProgramPage;
-  public int beginData;
-  public List<Integer> pageBuffers;
+  public long beginData;
+  public List<Long> pageBuffers;
   public int beginStack;
```

Comment:
These variables will receive an address. Using int might result in a sign extension error.